### DISCUSSION TITLE
技术预览版 - 2

### DISCUSSION THREAD

**Discussion by Richasy** [Richasy/Bili.Uwp#140] [通知]
包含以下修复：

- [替换滚动视图](https://github.com/Richasy/Bili.Uwp/pull/104)
- [处理全窗口状态下的播放器圆角问题](https://github.com/Richasy/Bili.Uwp/pull/125)
- [添加对播放失败状态的监听和状态处理](https://github.com/Richasy/Bili.Uwp/pull/135)
- [修复播放界面的UI问题](https://github.com/Richasy/Bili.Uwp/pull/137)
- [支持评论详情和视频标题选中](https://github.com/Richasy/Bili.Uwp/pull/138)

<hr /><em>This discussion was created from the release <a href='https://github.com/Richasy/Bili.Uwp/releases/tag/v2.2109.1901.0.pre-release'>技术预览版 - 2</a>.</em>

**Comment by LProphet** [comment]
牛啊

**Comment by Stapxs** [comment]
好耶！

**Comment by JustinXY** [comment]
我是win11 22458 专工版，播放时视频会卡死

**Reply by JustinXY** [reply]
64位的

**Reply by Stapxs** [reply]
同版本最新 dev 通道 win11
没遇到
可以试试设置里换个解码方式

**Reply by tm0210** [reply]
我也是22458,表示播放很流畅

**Comment by JustinXY** [comment]
这个蛮奇怪的，我已经尝试过更改解码，依旧没什么用，是不是A卡的问题，我是5700XT

发自我的iPhone

在 2021年9月19日，23:49，Stapx Steve ***@***.***> 写道：

﻿

同版本最新 dev 通道 win11
没遇到
可以试试设置里换个解码方式

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub<https://github.com/Richasy/Bili.Uwp/discussions/140#discussioncomment-1353985>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AVWOIKPU4DD2SCZNSQT44ELUCYBA5ANCNFSM5EKOIU5A>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Reply by Stapxs** [reply]
那不清楚
邮件回复没回复到子帖子下去挺怪的（

**Comment by JustinXY** [comment]
嗯，等等看作者的回复吧

发自我的iPhone

在 2021年9月20日，00:00，Stapx Steve ***@***.***> 写道：

﻿

那不清楚
邮件回复没回复到子帖子下去挺怪的（

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub<https://github.com/Richasy/Bili.Uwp/discussions/140#discussioncomment-1354022>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AVWOIKJK2NLKBHJ6PKBZR4TUCYCK3ANCNFSM5EKOIU5A>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Comment by Richasy** [comment]
@JustinXY 可以提交Issue，给准确的复现步骤和日志记录。我先看看能不能升到你的那个系统版本

**Comment by JustinXY** [comment]
谢谢，那我估计是显卡设置问题，我下班调试下

发自我的iPhone

在 2021年9月20日，11:33，tm0210 ***@***.***> 写道：

﻿

我也是22458,表示播放很流畅

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/Richasy/Bili.Uwp/discussions/140#discussioncomment-1355266>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AVWOIKPW5C5OE6OOGRMOVLTUC2TOVANCNFSM5EKOIU5A>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Comment by yoonawssf** [comment]
安装失败![IMG20210920150045.jpg](https://user-images.githubusercontent.com/90948695/133970585-61ab7043-1384-473f-af7d-87c33698af47.jpg)

**Reply by yoonawssf** [reply]
![RA)2AW5HK_TS53X0PIZ]77O.png](https://user-images.githubusercontent.com/90948695/133970809-fb407fe0-6f29-462a-9aaa-48bc821dfa91.png)

**Reply by Richasy** [reply]
下载的是符合设备架构的包吗？

**Reply by yoonawssf** [reply]
> 下载的是符合设备架构的包吗？

是的，所有版本我都试过了，都是这样，win11

**Comment by JustinXY** [comment]
我测试过了，确实显卡的设置问题，把增强垂直同步关了就好了，不过这个版本不能看大会员的清晰度了，您之前的那版倒是可以的
